tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -1130,17 +1171,24 @@ def forward(
                     max_active_clusters,
                     stream,
                     False,  # swap_ab=False for SwiGLU
+                ]
+                compile_kwargs = dict(
                     options=f"--opt-level 2 --enable-tvm-ffi"
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)